index.js

```diff
@@ -132,7 +132,7 @@ const runAction = () => {
 			run(
 				`${useNpm ? "npx --no-install" : "yarn run"} ${cmd} --${platform} ${
 					release ? "--publish always" : ""
-				} ${args}`,
+				} ${(platform == "mac") ? "--arm64 --x64" : ""}	${args}`,
```

Comment:
Since you can configure a universal build as well should this be something that is flagged?

Reply:
Actually that’s not a bad idea. I’ll look into it…
`- Yan`

Reply:
```yaml
strategy:
      matrix:
        os: [macos-latest, windows-latest]
        arch: ["--x64", "--arm64"]

- name: Build/release Electron app
        uses: samuelmeuli/action-electron-builder@v1
        with:
          args: ${{ matrix.arch }}
```